cmake/Modules/FindJack.cmake

```diff
@@ -44,10 +45,15 @@ else (JACK_LIBRARIES AND JACK_INCLUDE_DIRS)
     NAMES
       jack
     PATHS
+      ${_JACK_LIBRARY_DIRS}
       ${_JACK_LIBDIR}
+      /usr/lib64
       /usr/lib
+      /usr/local/lib64
       /usr/local/lib
+      /opt/local/lib64
       /opt/local/lib
+      /sw/lib64
       /sw/lib
```

Comment:
Normally, this is is not necessary since `lib` on 64bit systems is symlinked to `lib64`. Then `lib` and `lib64` contain the same things.

Reply:
Not true. On standard FHS, `lib64` is 64-bit libraries, and `lib` is 32-bit libraries. On some distros, `lib` is 64-bit libraries, and `lib32` is 32-bit libraries. On _others_, it's as you said.

Reply:
`lib` is for the main arch of the system and required. If this system is 64bit, `lib64` is optional and `lib32` is needed if you install 32bit lib on your 64bit system.

https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.pdf

Reply:
While it's interesting that the doc says that now (and I'd like to fix this someday for sure!), right now, pretty much all RPM based distributions use the scheme I describe, and both Arch and Gentoo design their FHS to be compatible with this style.

Reply:
I can confirm that in ArchLinux, `lib64` is a sysmlink of `lib` and add `lib32` when we use multilib repo which give access to 32bit libraries.

Reply:
Do you want me to add `/usr/lib32`? I can if you want...

Reply:
No, I just explained how ArchLinux manage `lib<qual>` folders.

In fact there is no need to add `lib64` since `lib` should be the right one for the system.

Reply:
It's wrong for Fedora, which is why I added it.

Reply:
If RPM based distros work like this it means (which is quite messy for my point of view), that PR is not a fix to use PipeWire's libjack but make it able to find JACK lib on 64bit RPM based distros.

Reply:
Well, it finds libjack as long as it's in the default library path that is set by the `%cmake` macro used in RPM packaging, yes. But otherwise it would not find it at all. It definitely doesn't work regardless of distribution if libjack is outside of the normal library search path (which is how I discovered this problem).

Reply:
I mean in other distros `lib` and `lib64` are the same so searching only in `lib` is not a issue but in RPM based searching in `lib64` is needed.

Reply:
Sure, but this also makes it so that libjack in non-default directories can actually be used because the directory path from pkgconfig is actually propagated to the hint list.